adafruit_trellism4.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, pin, *, width, height, rotation=0):
             width, height = height, width
         self._width = width
         self._height = height
+        self._auto_write = True
 
```

Comment:
If this is the desired state why not just set auto_wite=True in the _init  above?